### ISSUE TITLE
Apply to be a Digital Public Good

### ISSUE THREAD

**Issue by mgifford** [openemr/openemr#7877]
### Describe the problem

It would be great if OpenEMR was a Digital Public Good:

https://www.digitalpublicgoods.net/

I really do think that there is an alignment to the UN's SDG:

https://sdgs.un.org/goals

### Describe the solution you'd like

I'd love to see OpenEMR get involved https://www.digitalpublicgoods.net/get-involved

**Comment by adunsulag** [comment]
@mgifford I agree that we align pretty well with a number of the goals there.  I looked through the 7 screening questions and looking at the questions I think we'd need to clarify what is meant by the clear ownership requirement.  As OpenEMR is an open-source community the code is pretty much owned by all the contributors who have contributed to the project over the years.  I'm not sure if that meets their criteria, or if there needs to be a single clear owner such as some projects have with a Contributors License Agreement to transfer over their code ownership to the project.

Another question is, also how DPG will consider the tracking of PII and how that relates to a medical practice management software as the entire system is about tracking PHI.

**Comment by stephenwaite** [comment]
thanks for bringing this to our attention @mgifford! we can bring this up at a future openemr foundation meeting and can also discuss on the community forum to reach a wider audience

[community forum thread](https://community.open-emr.org/t/should-openemr-apply-to-be-a-digital-public-good/26525)
